README.md

```diff
@@ -73,19 +73,37 @@ changes, the default serialization format is the oldest one.
 - `bincode_1_3`: [bincode](https://docs.rs/bincode/1.3.3/bincode/) v1.3 (default)
 - `bincode_2_rc`: [bincode](https://docs.rs/bincode/2.0.0-rc.3/bincode/) v2.0.0-rc3
 - `postcard_1_0`: [postcard](https://docs.rs/postcard/1.0.0/postcard/) v1.0
+- `rpm_serde_1_3`: [rmp-serde](https://docs.rs/rmp-serde/1.3.0/rmp_serde/) v1.3
 
 ### Custom serialization format
 
 Define a struct with the name you want. This struct must implement [`native_model::Encode`](https://docs.rs/native_model/latest/native_model/trait.Encode.html) and [`native_model::Decode`](https://docs.rs/native_model/latest/native_model/trait.Decode.html) traits.
 
-Full examples: 
+Full examples:
 - [bincode with encode/decode](./tests_crate/tests/example/custom_codec/bincode.rs)
 - [bincode with serde](./tests_crate/tests/example/custom_codec/bincode_serde.rs)
 
 Others examples,  see the default implementations:
 - [bincode v1.3](./src/codec/bincode_1_3.rs)
--  [bincode v2.0 (rc)](./src/codec/bincode_2_rc.rs)
--  [postcard v1.0](./src/codec/postcard_1_0.rs)
+- [bincode v2.0 (rc)](./src/codec/bincode_2_rc.rs)
+- [postcard v1.0](./src/codec/postcard_1_0.rs)
+- [rmp-serde v1.3](./src/codec/rmp_serde_1_3.rs)
+
+### Notice
+`native_model` provides implementations that rely on metadata-less formats and `serde`.
+There are known issues with some `serde` advanced features such as:
```

Comment:
@lu-zero Is it possible to indicate that this is related to the use of certain `bincode` or other serializers that have a format without metadata ? And that it is possible to use other formats that include metadata.

Reply:
Right now all the impl in there have the problem, with __maybe__ rmp-serde exempt even if it is not clear in their documentation and I'd need to check how exactly they encode structs with `skip_serializing_if`, but the default configuration states it encodes structs as tuples.

Reply:
Potentially we can have a rmp-serde with a configuration that __surely__ works with all the serde features.

Reply:
Potentially we could provide also the safer variant, is it worth it?

Reply:
> Potentially we can have a rmp-serde with a configuration that surely works with all the serde features.
> 
> Potentially we could provide also the safer variant, is it worth it?

@lu-zero Indeed, this could be the subject of a separate issue. For now, only `rmp-serde` seems okay to me.

In my opinion, a test suite will be necessary to understand what is provided, in order to truly know what is possible or not. This will help avoid regressions.